recipes/installer.rb

```diff
@@ -22,6 +22,7 @@
 remote_file '/opt/aws/cloudwatch/awslogs-agent-setup.py' do
   source 'https://s3.amazonaws.com/aws-cloudwatch/downloads/latest/awslogs-agent-setup.py'
   mode '0755'
+  action :create_if_missing
```

Comment:
i like this change, the only thing I worry about is that this prevents updates, and will cause version drift between instances. I don't currently know a better way to support both updates and version locking (as there is no repository with previous versions)